source/common/tcp/original_conn_pool.cc

```diff
@@ -38,6 +38,7 @@ OriginalConnPoolImpl::~OriginalConnPoolImpl() {
 }
 
 void OriginalConnPoolImpl::drainConnections() {
+  ENVOY_LOG(debug, "draining connections");
```

Comment:
optional, given we had a crash report with original conn pool, what do you think of just not doing draining for it and commenting that it's not supported for legacy code? If I were you I'd rather not support legacy code if I'd be on the hook for debugging it, but totally up to you :-P

Reply:
I'm inclined to leave it as-is until we find an issue specifically with the original pool. The crash report we had was not tested with the new pool yet. The logic to figure out whether we're using the old or new impl, in the place where we need to know (due to the ProdFactory vs ClusterManager stuff) is not trivial.